src/wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -1278,20 +1293,32 @@ public static function get_filename_from_disposition( $disposition_header ) {
 	 * Retrieves the query params for collections of attachments.
 	 *
 	 * @since 4.7.0
+	 * @since 6.9.0 Extends the `media_type` parameter to support filtering by multiple media types.
 	 *
 	 * @return array Query parameters for the attachment collection as an array.
 	 */
 	public function get_collection_params() {
 		$params                            = parent::get_collection_params();
 		$params['status']['default']       = 'inherit';
 		$params['status']['items']['enum'] = array( 'inherit', 'private', 'trash' );
-		$media_types                       = $this->get_media_types();
+		$media_types                       = array_keys( $this->get_media_types() );
 
 		$params['media_type'] = array(
 			'default'     => null,
-			'description' => __( 'Limit result set to attachments of a particular media type.' ),
-			'type'        => 'string',
-			'enum'        => array_keys( $media_types ),
+			'description' => __( 'Limit result set to attachments of a particular media type or media types.' ),
+			'oneOf'       => array(
```

Comment:
Array may already accept strings so oneOf and string type may not be needed? Not sure about this, please check or maybe @TimothyBJacobs can confirm.

Reply:
Thanks @adamsilverstein 🙇🏻 

I think https://github.com/WordPress/wordpress-develop/pull/9211 might be the one to follow since it was up first and takes care of `mime_types` too.

I'll update https://github.com/WordPress/wordpress-develop/pull/9211 as required if the author can't.

I forgot to close this one, sorry!

Reply:
> Array may already accept strings so oneOf and string type may not be needed?

The type has changed to `'type'        => array( 'string', 'array' ),` for simplification. Is that okay?

Reply:
Ah yeah, that works great. It looks like `rest_sanitize_array` calls `wp_parse_list`, which always processes comma-separated strings into arrays during sanitization.

So this will coerce the value to an array. I'll update. 

Thanks for nudging me in that direction.

Reply:
I appreciate the help on this one, folks! 🙇🏻